params/safety_config.yaml

```diff
@@ -0,0 +1,11 @@
+safety_bonds:
```

Comment:
Problem solved: rosparam has a delete command which is guaranteed to run before the other stuff in the launch file for some reason